light-client/eth-light-client/src/client.rs

```diff
@@ -0,0 +1,67 @@
+use std::{
+	path::PathBuf,
+	process::exit,
+	sync::{
+		atomic::{AtomicBool, Ordering},
+		Arc,
+	},
+};
+
+use ethers::types::Address;
+use eyre::Result;
+use helios::{client::ClientBuilder, config::networks::Network, prelude::*};
+
+use crate::{config::Config, db::DB};
+
+pub async fn start_client(config: Config, db: DB, term: Arc<AtomicBool>) -> Result<()> {
+	let mut client: Client<FileDB> = ClientBuilder::new()
+		.network(Network::MAINNET)
```

Comment:
We probably want to be able to configure this to listen to testnets...

Reply:
Add support of environment variable `NETWORK`.
```
NETWORK = "goerli" | "mainnet"
```